### ISSUE TITLE
Feature: Macros for problem file initial state generation

### ISSUE THREAD

**Issue by jan-dolejsi** [jan-dolejsi/vscode-pddl#6]
Here’s a feature that would be really great to have. Suppose we want a `next` relation on 100 things. Then we have to write:

 
```
(next c0 c1)
….
(next c99 c100)
```
All explicitly.

Suppose instead the modeller writes:
 
```
{sequence next c0 c100}
```
 

And the editor produces:

 
```
(next c0 c1)
(next c1 c2)
(next c2 c3)
```
Etc.

The general format is
 
`{sequence relation start end}`.

**Comment by jan-dolejsi** [comment]
Ok, I intend to solve that by a smart snippet that would generate the PDDL text and insert it into problem file.

**Comment by jan-dolejsi** [comment]
Released in v2.4.0
